docs/binary.md

```diff
@@ -291,9 +291,10 @@ Int value |  2 or 3 |    L    |
 
 Zero is always stored as positive; negative zero is illegal.
 
-If the value is zero then _T_ must be 2, _L_ is zero, and there are no length or
-magnitude subfields. As a result, when _T_ is 3, both _L_ and the magnitude
-subfield must be non-zero.
+If the value is zero then _T_ must be 2, _L_ should be zero, and if _L_ is zero 
+there must be no length or magnitude subfields. However, implementations should 
```

Comment:
As discussed in #93, allowing for inconsistent handling of this case between implementations is probably not justified. Let's remove the second sentence altogether, resulting in:

> If the value is zero then _T_ must be 2, _L_ should be zero, and if _L_ is zero there must be no length or magnitude subfields. When _T_ is 3, both _L_ and the magnitude subfield must be non-zero.